cmd/hyperkube/hyperkube_test.go

```diff
@@ -177,15 +177,15 @@ func TestServerHelp(t *testing.T) {
 	x := runFull(t, "hyperkube test1 --help")
 	assert.NoError(t, x.err)
 	assert.Contains(t, x.output, "A simple server named test1")
-	assert.Contains(t, x.output, "--help[=false]: help for hyperkube")
+	assert.Contains(t, x.output, "-h, --help                                               help for hyperkube")
 	assert.NotContains(t, x.output, "test1 Run")
 }
 
 func TestServerFlagsBad(t *testing.T) {
 	x := runFull(t, "hyperkube test1 --bad-flag")
 	assert.EqualError(t, x.err, "unknown flag: --bad-flag")
 	assert.Contains(t, x.output, "A simple server named test1")
-	assert.Contains(t, x.output, "--help[=false]: help for hyperkube")
+	assert.Contains(t, x.output, "-h, --help                                               help for hyperkube")
 	assert.NotContains(t, x.output, "test1 Run")
```

Comment:
Yes. Here you can find example(several options) of old help:
```
      --global-housekeeping-interval=1m0s: Interval between global housekeepings
      --google-json-key="": The Google Cloud Platform Service Account JSON Key to use for authentication.
      --healthz-bind-address=127.0.0.1: The IP address for the health check server to serve on, defaulting to 127.0.0.1 (set to 0.0.0.0 for all interfaces)
      --healthz-port=10249: The port to bind the health check server. Use 0 to disable.
  -h, --help[=false]: help for hyperkube
      --hostname-override="": If non-empty, will use this string as identification instead of the actual hostname.
      --housekeeping-interval=10s: Interval between container housekeepings
      --iptables-masquerade-bit=14: If using the pure iptables proxy, the bit of the fwmark space to mark packets requiring SNAT with.  Must be within the range [0, 31].
      --iptables-sync-period=30s: How often iptables rules are refreshed (e.g. '5s', '1m', '2h22m').  Must be greater than 0.
```

after this patch it will look like:
[      --global-housekeeping-interval value                 Interval between global housekeepings (default 1m0s)
      --google-json-key string                             The Google Cloud Platform Service Account JSON Key to use for authentication.
      --healthz-bind-address value                         The IP address for the health check server to serve on, defaulting to 127.0.0.1 (set to 0.0.0.0 for all interfaces) (default 127.0.0.1)
      --healthz-port value                                 The port to bind the health check server. Use 0 to disable. (default 10249)
  -h, --help                                               help for hyperkube
      --hostname-override string                           If non-empty, will use this string as identification instead of the actual hostname.
      --housekeeping-interval value                        Interval between container housekeepings (default 10s)
      --iptables-masquerade-bit value                      If using the pure iptables proxy, the bit of the fwmark space to mark packets requiring SNAT with.  Must be within the range [0, 31]. (default 14)
      --iptables-sync-period duration                      How often iptables rules are refreshed (e.g. '5s', '1m', '2h22m').  Must be greater than 0. (default 30s)
](url)